src/Symfony/Component/Notifier/Bridge/Slack/SlackTransportFactory.php

```diff
@@ -28,17 +29,21 @@ final class SlackTransportFactory extends AbstractTransportFactory
      */
     public function create(Dsn $dsn): TransportInterface
     {
-        $scheme = $dsn->getScheme();
+        if ('slack' !== $dsn->getScheme()) {
+            throw new UnsupportedSchemeException($dsn, 'slack', $this->getSupportedSchemes());
+        }
+
+        if ('/' !== $dsn->getPath()) {
+            $message = sprintf('Invalid "%s" notifier DSN: Using a dsn for slack webhooks is deprecated (maybe you haven\'t update the DSN when upgrading from 5.1).', $dsn->getOriginalDsn());
```

Comment:
Wording: "deprecated" means that the feature is still available, but we will stop supporting it in the future. This is not the case here, because the feature has actually been removed already.

Also, please inline `$message`.

Reply:
What do you think now? My english isn't very well ^^